customer-backend/src/main/java/io/eventuate/examples/tram/ordersandcustomers/customers/CustomerConfiguration.java

```diff
@@ -1,22 +1,25 @@
 package io.eventuate.examples.tram.ordersandcustomers.customers;
 
+import io.eventuate.examples.tram.ordersandcustomers.customers.service.CustomerOptimisticLockingDecorator;
 import io.eventuate.examples.tram.ordersandcustomers.customers.service.CustomerService;
 import io.eventuate.examples.tram.ordersandcustomers.customers.service.OrderEventConsumer;
-import io.eventuate.tram.consumer.kafka.TramConsumerKafkaConfiguration;
+import io.eventuate.jdbckafka.TramJdbcKafkaConfiguration;
```

Comment:
`io.eventuate.jdbckafka` is clearly a badly named package.
It should at least be `eventuate.tram.jdbckafka`
(It includes consumer/producer dependencies, so io.eventuate.tram.consumer wasn't quite right)

Reply:
I will fix, but as I remember I did not change package name.
I think it was from beginning.